git-mktree: add page (#11725)
tldr-pages/tldr: pages/common/git-mktree.md

# git mktree

> Build a tree-object using ls-tree formatted text.
> More information: <https://git-scm.com/docs/git-mktree>.

> - Allow for missing objects which bypass the default behavior that verifies each tree entry has an existing object:

Reviewer: ```suggestion
- Allow missing objects which bypass the default behavior that verifies each tree entry has an existing object:
```

> 
> - Allow missing objects which bypass the default behavior that verifies each tree entry has an existing object:
> 
> `git mktree --missing`

Reviewer 2: ```suggestion

- Build a tree object and verify that each tree entry’s hash identifies an existing object:

`git mktree`

- Allow missing objects:

`git mktree --missing`
```

> - Read the NUL terminated output of the tree object:

Reviewer 2: ```suggestion
- Read the NUL terminated output of the tree object (`ls-tree -z`):
```

`git mktree -z`

> - Allow for the creation of multiple tree objects:

Reviewer: ```suggestion
- Allow creation of multiple tree objects:
```
Maybe we can simplify this description?

> `git mktree --batch`

Reviewer 3: ```suggestion

- Sort and build a tree from `stdin` (non-recursive `git ls-tree` output format is required):

`git mktree < {{path/to/file}}`
```
